image/activate_func.sh

```diff
@@ -16,7 +16,7 @@ function activate_holy_build_box_deps_installation_environment() {
 	export CPPFLAGS=-I/hbb/include
 	export LDPATHFLAGS="-L/hbb/lib -Wl,-rpath,/hbb/lib"
 	export LDFLAGS="$LDPATHFLAGS"
-	export LD_LIBRARY_PATH=/hbb/lib
+	export LD_LIBRARY_PATH=/hbb/lib:${LD_LIBRARY_PATH}
```

Comment:
You should put this assignment inside quotes, like this:

    export LD_LIBRARY_PATH="/hbb/lib:${LD_LIBRARY_PATH}"

Otherwise bash won't handle things properly if the old LD_LIBRARY_PATH contains special characters, like multiple spaces.